apps/studio/src/server/modules/resource/resource.service.ts

```diff
@@ -134,7 +140,8 @@ export const getPageById = (
       ]),
     )
     .select(defaultResourceSelect)
-    .executeTakeFirstOrThrow()
+    .executeTakeFirst()
```

Comment:
why do we not opt to throw here? wouldn't this be a 404?

Reply:
I waffling between trpc errors being thrown inside `service` files or in the router files directly. Might want the caller to take responsibility instead (which the caller actually does in this case)